server/src/main/java/org/opensearch/search/GenericSearchExtBuilder.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2023 Aryn
```

Comment:
Please use the our standard header:

```
/*
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 */

```

Generally, we  attribute the  copyright to `OpenSearch Contributors` (who you definitely are)